docker/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2'
 
 services:
   master:
-    image: chrislusf/seaweedfs
+    build: .
```

Comment:
maybe put a comment here to show the different options, either build local or use the image. same for other 2 places.

Reply:
Added comments :)